pkg/dbal/DbalConsumer.php

```diff
@@ -176,7 +176,7 @@ protected function receiveMessage()
                 ->where('queue = :queue')
                 ->andWhere('(delayed_until IS NULL OR delayed_until <= :delayedUntil)')
                 ->orderBy('priority', 'desc')
-                ->orderBy('id', 'asc')
+                ->addOrderBy('id', 'asc')
```

Comment:
it should be `desc`. gonna fix it

Reply:
for ref https://github.com/php-enqueue/enqueue-dev/pull/198/files#diff-12c1fdd4a765fda7186569d999b34749R179